spec/ghp_metadata_generator_spec.rb

```diff
@@ -18,10 +18,29 @@
       ENV['PAGES_REPO_NWO'] = nil
     end
 
-    it "raises a NoRepositoryError" do
-      expect(-> {
-        subject.send(:nwo, site)
-      }).to raise_error(Jekyll::GitHubMetadata::NoRepositoryError)
+    context "without a git nwo" do
+      it "raises a NoRepositoryError" do
+        allow(subject).to receive(:git_remote_url).and_return("")
+        expect(-> {
+          subject.send(:nwo, site)
+        }).to raise_error(Jekyll::GitHubMetadata::NoRepositoryError)
+      end
+    end
+
+    it "retrieves the git remote" do
+      expect(subject.send(:git_remote_url)).to include("jekyll/github-metadata")
```

Comment:
This seems to fail. Think it's hitting your stub above?

https://travis-ci.org/jekyll/github-metadata/jobs/117149521#L232